tests/misc/test_basic_without_ssh.py

```diff
@@ -104,7 +104,3 @@ def live_migrate(vm, dest_host, dest_sr, check_vdis=False):
         else:
             logging.info("* Preparing for live migration without storage motion *")
             live_migrate(vm, host1, existing_shared_sr)
-
-    def test_shutdown(self, imported_vm):
-        vm = imported_vm
-        vm.shutdown(verify=True)
```

Comment:
As the file was formulated that test was depending on rather unrelated ones, just so they can run before the VM is shutdown, so to those tests this (as well as the VM start BTW) is/should-be really a fixture.

Now for explicit testing that's different. We can have a separate test focusing on start+shutdown (since it makes little sense to be able to run those as separate tests), and leave the rest letting the start/shutdown handled by a fixture.

Reply:
done